engine/src/button.cpp

```diff
@@ -2603,11 +2613,14 @@ Boolean MCButton::findmenu(bool p_just_for_accel)
 				MCU_set_rect(trect, 0, 0, rect.width, height + 4);
 				trect = MCU_reduce_rect(trect, MClook == LF_MOTIF ? DEFAULT_BORDER : 1);
 				fptr->setrect(trect);
-				/* UNCHECKED */ MCStackSecurityCreateStack(menu);
-
-				menu->setparent(this);
-				menu->createmenu(fptr, rect.width, height + 4);
-				MCdispatcher->appendpanel(menu);
+                MCStack * t_menu;
+				/* UNCHECKED */ MCStackSecurityCreateStack(t_menu);
```

Comment:
**dons @peter-b hat** This could probably `return False` here if creating the stack fails, couldn't it?